openapi.yml

```diff
@@ -2575,6 +2577,219 @@ paths:
                     type: string
                 in: path
                 required: true
+    /audit_logs:
+        summary: Path used to manage the list of AuditLogs.
+        description: >-
+            The REST endpoint/path used to list zero or more `AuditLog` entities.  This path contains a `GET`
+            operation to perform the list task.
+        get:
+            tags:
+                - AuditLogs
+            parameters:
+                -
+                    name: include
```

Comment:
Do AuditLogs have any relationships that we're going to show in the serializer?

Reply:
I don't think so. They're pretty standalone.

Reply:
Should remove the `include` param then